harness-delegate-ng/values.yaml

```diff
@@ -88,11 +88,14 @@ description: ""
 tags: ""
 
 # Permissions for installed delegate, could be CLUSTER_ADMIN, CLUSTER_VIEWER or NAMESPACE_ADMIN
```

Comment:
could you help make this comment more clear
Set delegate to use the default cluster roles based on 'k8sPermissionsType'